src/pages/gd/Stake/index.tsx

```diff
@@ -499,7 +499,20 @@ export default function Stakes(): JSX.Element | null {
                     )}
                 </Modal>
             </StakesSC>
-            {network !== 'production' ? <Savings /> : <></>}
+            <>
```

Comment:
this looks like list in [gd/Portfolio/index.tsx](https://github.com/GoodDollar/GoodProtocolUI/pull/246/files/fbe83765ae5e03552c88423a05d3cc4ae5fd2baf..72f848bb9dba95bc87fd3895aed477a64b7e8cf1#diff-416d64617fe8aa048d9beab9775979fb5e7c09266bcff94ae5db27de85cb4025)

could it be reused / created common component reused in the both files ?